server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationIT.java

```diff
@@ -833,4 +861,207 @@ public void testPressureServiceStats() throws Exception {
             });
         }
     }
+
```

Comment:
nit simple java docs,

```suggestion
/** tests a scroll query on the replica */
```